packages/modules/src/modules/epics/ContributionsEpic.tsx

```diff
@@ -260,21 +280,35 @@ const ContributionsEpic: React.FC<EpicProps> = ({
     hasConsentForArticleCount,
     stage,
 }: EpicProps) => {
-    const countryGroupId = countryCodeToCountryGroupId(countryCode || 'GBPCountries');
-    const defaultFrequency: ContributionFrequency = variant.defaultChoiceCardFrequency || 'MONTHLY';
-    const [choiceCardSelection, setChoiceCardSelection] = useState<ChoiceCardSelection | undefined>(
-        variant.choiceCardAmounts && {
-            frequency: defaultFrequency,
-            amount:
-                variant.choiceCardAmounts[countryGroupId]['control'][defaultFrequency][
-                    'amounts'
-                ][1],
-        },
-    );
+    const { image, tickerSettings, showChoiceCards, choiceCardAmounts } = variant;
 
-    const { hasOptedOut, onArticleCountOptIn, onArticleCountOptOut } = useArticleCountOptOut();
+    const currencySymbol = getLocalCurrencySymbol(countryCode);
 
-    const { image, tickerSettings, showChoiceCards, choiceCardAmounts } = variant;
+    const [choiceCardSelection, setChoiceCardSelection] = useState<
+        ChoiceCardSelection | undefined
+    >();
+
+    const [amountsVariant, setAmountsVariant] = useState<SelectedAmountsVariant | undefined>();
```

Comment:
The `amountsVariant` state is unnecessary here and can be removed - you can just use `choiceCardAmounts` directly

Reply:
I'll give it a go

Reply:
Done